oda_api/api.py

```diff
@@ -841,9 +841,9 @@ def get_product(self,
                         msg += '%s' % valid_names
                         msg += '\n'
                         msg += 'this will throw an error in a future version \n'
-                        msg += 'and might breack the current request!\n '
+                        msg += 'and might brake the current request!\n '
                         msg += '----------------------------------------------------------------------------\n'
-                        warnings.warn(msg)
```

Comment:
this is actually a warning. And, following note above, not `logging.warn` bit `warnings.warn`

Reply:
According to what we discussed in #59 , I thought this was supposed to be changed to debug level message. Or perhaps I misunderstood.

Reply:
> According to what we discussed in #59 , I thought this was supposed to be changed to a debug level message. Or perhaps I misunderstood.

The part that lead to `unable to dig list, instance not a dict by...` should be.  This is utterly incomprehensible for the user.

`invalid parameter` warning should remain. You can imagine it is important that user is warned (at least!) when using incorrect parameter name.
It should be separately determined why, as you noted as well, `T_format` is not understood. It should be, in principle, understood and used correctly. Maybe not in all queries though.
I guess it was passed by frontend API.
In any case, at this level, it should be clear to user that this parameter is not used.

Reply:
Ok, now is clear. I was confused and thought we were referring to the warning message that should have been changed.

As of `T_format`, I had indeed the same impression.